widgets/blueprints/src/widget.tsx

```diff
@@ -85,26 +95,20 @@ Stage.defineWidget<unknown, unknown, BlueprintsWidgetConfiguration>({
             });
         }
 
-        return searchActions
-            .doListBlueprints(filterRules, {
-                _include: 'id,updated_at,created_at,description,created_by,visibility,main_file_name,state,error',
-                ...params
-            })
-            .then(data => {
-                result.blueprints = data;
-
-                return toolbox.getManager().doGetFull('/summary/deployments', {
-                    _target_field: 'blueprint_id',
-                    blueprint_id: _.map(data.items, item => item.id)
-                });
-            })
-            .then(data => {
-                result.deployments = data;
-                return result;
-            });
+        const blueprintsList = await searchActions.doListBlueprints(filterRules, {
+            _include: 'id,updated_at,created_at,description,created_by,visibility,main_file_name,state,error',
+            ...(params as ListBlueprintsParams)
```

Comment:
```suggestion
            ...params
```

Seems like usage of `ListBlueprintsParams` type is not necessary here
After removing it, it would be also required to remove `ListBlueprintsParams` type import from the beginning of the file